prepublish.ts

```diff
@@ -0,0 +1,48 @@
+import * as fs from 'fs'
+import * as https from 'https'
+import * as path from 'path'
+
+const UPSTREAM_COMMIT_HASH = '8c762559c98b707801f52dd070dd39ab9478b876'
+const UPSTREAM_REPO = 'https://raw.githubusercontent.com/cowprotocol/composable-cow'
+const ABI_TO_FETCH = ['ComposableCoW', 'ExtensibleFallbackHandler', 'TWAP']
```

Comment:
https://raw.githubusercontent.com/cowprotocol/composable-cow/8c762559c98b707801f52dd070dd39ab9478b876/out/ComposableCoW.sol/ComposableCoW.json

The file contains lots of redundant code like:

<img width="1694" alt="image" src="https://github.com/cowprotocol/cow-sdk/assets/7122625/258b7014-dc0a-4c87-885c-b401a2856803">

Will it be added to the final bundle? If yes, then it will ad excessive size to frontend consumers.

Reply:
<img width="1023" alt="image" src="https://github.com/cowprotocol/cow-sdk/assets/7122625/2dcd5dd2-3fa0-4684-acf8-40b7cb52e222">

Yeah, it is in the bundle :(

Could you please trim API files and keep only useful data there?

Reply:
Would it be possible to load those artifacts async, only if needed to help trim down the bundle?